Tests/CollectionsTestSupportTests/CombinatoricsChecks.swift

```diff
@@ -14,7 +14,7 @@ import XCTest
 import _CollectionsTestSupport
 #endif
 
-class CombinatoricsTests: CollectionTestCase {
+final class CombinatoricsTests: CollectionTestCase {
```

Comment:
I'm not interested in microoptimizing XCTestCase classes. Please remove these changes from all test files.

Reply:
Thank you for the reply. i will delete the changes in the test files.